docs/api/index-patterns.asciidoc

```diff
@@ -0,0 +1,27 @@
+[[index-patterns-api]]
+== Index patterns APIs
+
+experimental[] Manage {kib} index patterns using REST API.
+
+WARNING: Do not write documents directly to the `.kibana` index. When you write directly
+to the `.kibana` index, the data becomes corrupted and permanently breaks future {kib} versions.
+
+WARNING: Prefer using index patterns API for managing {kib} index patterns instead of lower-level <<saved-objects-api, saved objects API>>.
```

Comment:
```suggestion
WARNING: Use the index patterns API for managing {kib} index patterns instead of lower-level <<saved-objects-api, saved objects API>>.
```